tools/cachemgr.cc

```diff
@@ -1252,7 +1241,7 @@ check_target_acl(const char *hostname, int port)
 
     if (fp == NULL) {
 #ifdef CACHEMGR_HOSTNAME_DEFINED
-
+        //TODO: simplify and maybe get rid of CACHEMGR_HOSTNAME altogether
```

Comment:
Whitespace nitpick:

```suggestion
        // TODO: simplify and maybe get rid of CACHEMGR_HOSTNAME altogether
```

I do not insist on this change.

Reply:
no prob, changing